torch/_inductor/codegen/wrapper.py

```diff
@@ -108,6 +122,11 @@ def get_cpp_op_schema(kernel):
 
 SUPPORTED_FALLBACK_CPP_WRAPPER = [
     "repeat_interleave.Tensor",
+    # these are supported kernels but can show up as fallbacks if inputs are unsupported dtypes.
+    # TODO: handle these more generally instead of hardcoding all of them here.
+    "convert_element_type.default",
+    "_fft_c2c.default",
+    "view_as_real.default",
```

Comment:
The aten ops should have hit this if-branch, https://github.com/pytorch/pytorch/blob/194262ee49961acc8d84d6d4672748eae1826c30/torch/_inductor/ir.py#LL3222C16-L3222C16 (likely because .default is not matched).

Reply:
Oh, thanks for the pointer - updated it to iteratively getattr if the `kernel.__name__` has a `.` in it